vmdb/app/controllers/ops_controller/db.rb

```diff
@@ -32,7 +32,7 @@ def db_list(exp=nil)
       end
       exp =  MiqExpression.new(condition)
     elsif model == VmdbDatabaseConnection
-      @zones = Zone.find(:all).sort_by(&:name).collect { |z| [z.name, z.name] }
+      @zones = Zone.all.sort_by(&:name).collect { |z| [z.name, z.name] }
```

Comment:
Can this just be turned into a pluck (or are you trying to avoid shiny objects :sparkles:)?  `Zone.pluck(:name).sort.collect { |z| [z, z] }`